src/components/client/campaign-application/CampaignApplicationPage.tsx

```diff
@@ -0,0 +1,5 @@
+import Layout from '../layout/Layout'
+
+export default function CampaignApplicationPage() {
+  return <Layout></Layout>
```

Comment:
This line doesn't pass the linter self-closing rule. Can you change it  to `<Layout/>` or add some child element to it.

Reply:
Made it self-closing as I have already started working on another branch where I am adding the children elements.